templates/Layout/DNRoot_alerts.ss

```diff
@@ -13,18 +13,15 @@
 
 <div class="content">
 
-	<% if $AlertsConfigContent %>
-		<h3>Currently configured alerts</h3>
-		<pre>$AlertsConfigContent</pre>
+	<% if $AlertsConfigContent(master) %>
```

Comment:
Can you confirm this doesn't double the requests, i.e. that is caches the first call?